recipe/meta.yaml

```diff
@@ -12,31 +12,32 @@ source:
 
 build:
   number: 8
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   skip: true  # [py<36]
 
 requirements:
   build:
-    - ninja
-    - cmake >=3.15
-    - make                                   # [unix]
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
     - {{ compiler('cxx') }}
```

Comment:
Not really needed, but with Cython you should also add `compiler('c')` to the build requirements. What we have right now works because it pulls in the C libs as well.

Reply:
We could go one step further and remove `cxx` and just keep `c`.